cjaasBrowserSDK/README.md

```diff
@@ -42,23 +47,290 @@ cjaas("streamReadToken", "SAS Token from CJaaS Account to read from stream");
 
 // page events will be tracked
 cjaas("option", "PAGE_TRACK", true);
+cjaas("option", "ENABLE_ORCHESTRATION", true);
+
+// init tracking
+cjaas("init");
+
+//Identify yourself
+cjaas("identify", `personID`, "Person Name"); 
+
+//sending data {"title": "Outliers", "price": 50} . Track event : User buys book
+cjaas("track", "User buys book", {"title": "Name_of_the_book", "price": 50});
+cjaas("track", "User buys book", {"title": "Name_of_the_book", "price": 70});
+```
+
+This sets up the cjaas to any website, and allows us to send sample events such as "User buys book" to CJaaS with Data such as "title" and "Price".
+After this we can create profile view template and journey action in CJaaS to trigger walkin. 
+
+Journey action has rules in it and when this rules get satisfied against the template, walkin gets triggered. 
+
+**Create Profile View Template:**
+
+POST https://cjaas_host_url/v1/journey/views/templates
+
+SAS Token Requirements:
+
+service = profile
+
+permission = w
+
+Headers(Mandatory)
+
+Authorization: SAS Signature
+
+Body
+
+```json
+
+  
+  {
+      "id": "buying pattern of Some Customer",
+      "namespace": "namespace_details",
+      "organization": "org_details",
+
+      "attributes": [
+          {
+              "version": "1.0",
+              "event": "User buys book",
+              "metadataType": "string",
+              "metadata": "title",
+              "limit": 100,
+              "displayName": "books bought",
+              "lookbackDurationType": "days",
+              "lookbackPeriod": 30,
+              "aggregationMode": "Value",
+              "verbose": true
+          },
+          {
+
+              "version": "1.0",
+              "event": "User buys book",
+              "metadataType": "string",
+              "metadata": "title",
+              "limit": 100,
+              "displayName": "Number of books bought",
+              "lookbackDurationType": "days",
+              "lookbackPeriod": 30,
+              "aggregationMode": "Count",
+              "verbose": true
+          },
+          {
+
+              "version": "1.0",
+              "event": "User buys book",
+              "metadataType": "integer",
+              "metadata": "price",
+              "limit": 100,
+              "displayName": "Total amount spent",
+              "lookbackDurationType": "days",
+              "lookbackPeriod": 50,
+              "aggregationMode": "Sum",
+              "verbose": true
+          }
+      ]
+  }
```

Comment:
Let's keep our initial how-to very basic. By providing nested actions and multiple swim-lanes people will be confused.

Reply:
done